### PR TITLE
Team members updated and questions added

### PR DESCRIPTION
**Pull request by Mvrcusj**

### CODE REVIEW

File: QUESTIONS.md [hunk 1]
```diff
@@ -3,3 +3,11 @@
 What questions can you come up with?
 
 ## Cohort Questions
+
+1. Is "Time-Travel" exclusive to React? And is it possible to Time-Travel back to week 1 with my current knowledge so I know what I'm in for and can mentally prepare?!
```
Comment (crkingsley) [inline QUESTIONS.md:14]: +1

**Review by crkingsley — APPROVED** [review]
LGTM!!

File: QUESTIONS.md [hunk 2]
```diff
@@ -3,3 +3,11 @@
...
+
+2. Why is immutability important and is it's importance specific to React or is it as important for all of JS?
+
+3. Are React Devtools different than Chrome Devtools?
```
Comment (hollisjamison) [inline QUESTIONS.md:18]: Yes! It adds 2 new tabs to your Chrome dev tools. It's great for seeing props without having to console.log

**Review by SBucharelli — APPROVED** [review]

File: QUESTIONS.md (hunk 1 above)
Comment (AL743-ni82dy) [inline QUESTIONS.md:14]: LoL

File: QUESTIONS.md (hunk 2 above)
Comment (Mvrcusj) [reply QUESTIONS.md:18]: Awesome! I'll definitely look into it! Thanks!

**Pull request merged**